tests/lib/reset.sh

```diff
@@ -10,6 +10,20 @@ reset_classic() {
     # have changed on the disk.
     systemctl daemon-reload
 
+    if [[ "$SPREAD_SYSTEM" == ubuntu-14.04-* ]]; then
```

Comment:
Why is this special cased for Ubuntu 14.04?

Reply:
@zyga, I could not reproduce that error in other releases, also all the failures that we see are always happening just on 14.04., I guess it is a problem on systemd and just happening on that version. I can make it independent of the OS, that should not affect.

Reply:
I'd prefer if it was universally enabled as all the special casing makes it harder to keep a mental model of what is going on where. If it works let's enable it everywhere.

Reply:
@zyga addressed in the last commit, thanks for reviewing this.